ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -74,6 +74,7 @@ export const AccountListItem = ({
   isHidden = false,
   currentTabOrigin,
   isActive = false,
+  startAccessory,
```

Comment:
Should we default this to `null`?

Reply:
Here `startAccessory` must always receive a valid React node. Because we need to pass checkbox here. If I change it to null, I get TS errors in the ConnectAccountsModal where I am passing the checkbox